### PR TITLE
Fixes Issue 56

### PR DESCRIPTION
**Pull request by thenewguy**
Fixes Issue 56

https://github.com/PolicyStat/jobtastic/issues/56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,6 +349,16 @@ all of your arguments must be keyword arguments.
 Note: This is a limitation of the current `significant_kwargs` implementation,
 and totally fixable if someone wants to submit a pull request.
 
+### async_or_eager
```
Comment (winhamwr) [inline README.md]: :+1: 

Could we list these two below the `delay_or` versions? My impression is that most folks will continue to use the simpler signature of `delay_or`.

Comment (winhamwr) [inline README.md]: I think listing the two `async_` versions after the `delay_` versions still makes sense.

**Comment by thenewguy** [comment]
Fixed.  I thought I had already addressed your comments but we were waiting on me =P

**Comment by winhamwr** [comment]
Do you think it's worth adding a test on `async_or_FOO` that actually sets one of the advanced task options that we're trying to get access to and verifies that it was used?

**Comment by winhamwr** [comment]
Looking through the documentation of [result](http://docs.celeryproject.org/en/latest/reference/celery.result.html?highlight=result), I don't see anything that would change as a result of something we passed in to `apply_async`, so this might be tough to test. Merging as-is.

**Pull request merged**